### DISCUSSION TITLE
Will Spelljammer be added any time soon/is this still maintained?

### DISCUSSION THREAD

**Discussion by That1Rogue** [kinkofer/FightClub5eXML#205] [General]
See title.

**Comment by kinkofer** [comment]
Short answer is it won’t be added soon. The way I was getting the data has changed and I haven’t had time to support that change. I believe there’s another repo that’s adding data manually, but that hasn’t been the goal of my repo.  On Jul 18, 2023, at 5:47 PM, That1Rogue ***@***.***> wrote:﻿
See title.

**Reply by That1Rogue** [reply]
Could you possibly point me in the direction of that other repo?

**Comment by kinkofer** [comment]
https://github.com/ogre-project/FightClub5eXML
